src/peergos/server/corenode/NonWriteThroughCoreNode.java

```diff
@@ -13,23 +13,25 @@
 public class NonWriteThroughCoreNode implements CoreNode {
 
     private final CoreNode source;
-    private final CoreNode temp;
+    private final ContentAddressedStorage ipfs;
+    private final Map<String, List<UserPublicKeyLink>> tempChains;
+    private final Map<PublicKeyHash, String> tempReverse;
```

Comment:
not sure about the variable names

Reply:
temp because they cache all changes and are not committed.

Reply:
sure, but the reverse lookup especially could be renamed to better reflect the String is actually the username

Reply:
done